library/src/main/java/com/chuckerteam/chucker/api/ChuckerInterceptor.kt

```diff
@@ -15,6 +19,7 @@ import okhttp3.Response
 import okhttp3.ResponseBody
 import okio.Buffer
 import okio.GzipSource
+import okio.Okio
 
 private const val MAX_BLOB_SIZE = 1000_000L
```

Comment:
Seems that this const is unused, because the same is present in companion object.

Reply:
I didn't touch because it is removed in another PR but I'll remove it here after we find a solution to the main issue in this PR. :)